testing/README.md

```diff
@@ -48,7 +48,7 @@ type TestingApp interface {
 
 	// ibc-go additions
 	GetBaseApp() *baseapp.BaseApp
-	GetStakingKeeper() stakingkeeper.Keeper
+	GetIBCTestStakingKeeper() ibctestingtypes.StakingKeeper
```

Comment:
I honestly liked the name `GetStakingKeeper`, since the name of the type already indicates that this is for testing, so I consider that is redundant to add `Test` in the name of the function... But no problem if we decide to rename it.

Reply:
i renamed on request from @ValarDragon in the issue, i don't have a particularly strong opinion either way tho. I can see the value add in being explicit, but it is true that the type is also quite informative.

Reply:
I think now that the interface returns the `ibctestingtypes.StakingKeeper` it can be redundant. I do agree `GetStakingKeeper` is cleaner, but I understand this function will be implemented in `app.go` for a production chain. 

@ValarDragon do you still prefer `GetIBCTestStakingKeeper` given the new return value